# Require that every code blocks in YUIDoc comments has a type associated (`ember-internal/require-yuidoc-code-block-type`)

<!-- end auto-generated rule header -->

## Examples

```js
/**
 * See code below:
 *
 * ```js
 * good
 * ```
 *
 * ```
 * bad
 * ```
 */
```
